docs-src/modules/getting-started/pages/index.adoc

```diff
@@ -1,14 +1,143 @@
+Transforming and combining timelines allows you to intuitively express computations over events.
+
+****
+⭢ Read more about timelines in xref:overview:what-is-kaskada.adoc[]
+****
+
+With Kaskada, it's timelines all the way down - every operation's inputs and outputs are timelines. 
+When you make a query, the resulting timeline is output in one of two ways: as a _history_ or as a _snapshot_. 
+
+* A timeline *History* contains a value each time the timeline changes, and each row is associated with a different entity and point in time. 
+* A timeline *Snapshot* contains a value for each entity at the _same_ point in time; each row is associated with a different entity, but all rows are associated with the same time.
+
+This output may be written in different ways -- for example, they may be written to a Parquet file or sent as events to a stream.
+
+****
+⭢ Read more about converting to tables in xref:developing:queries.adoc#configuring-how-queries-are-computed[Queries]
+****
+
+== Kaskada's basic architecture
+
+Kaskada is implemented as a standalone service with two main components.
+
+* The *engine* is a stateless compute engine for transforming event data.
+* The *manager* is an (optionally) stateful API for communicating with clients.
+
+These components are distributed as standalone, precompiled binaries.
+The Kaskada service is accessed using a gRPC API, and multiple clients exist.
+You can access the API using the provided Python client or using the CLI binary.
+Kaskada can be configured to run as a remote service or as a local process.
+
+****
+⭢ Read more about installing Kaskada in xref:ROOT:installing.adoc[]
+****
+
+== Kaskada's data model
+
+Timelines begin with xref:developing:tables.adoc[tables].
+Tables are how Kaskada stores input events.
+Tables consist of multiple events, and each event is a value of the same xref:fenl:data-model.adoc[type].
+
+[%header,cols=3*]
+.Events in the `Purchase` table
+|===
+| time | entity | value
+| 8:20 | Alice | `{amount: 5}`
+| 9:45 | Alice | `{amount: 2}`
+| 2:10 | Alice | `{amount: 13}`
+| 8:52 | Alice | `{amount: 4}`
+|===
+
+When querying Kaskada, the contents of a table are interpreted as a xref:fenl:continuity.adoc[discrete timeline]: the value associated with each event corresponds to a value in the timeline.
+
+[stream_viz,name=purchase-timeline]
+.Discrete timeline describing the `Purchase` table
+....
+[
+    {
+        "label": "Purchase",
+        "kind": "discrete",
+        "data": [
+            [
+                {"t": 2, "v": "{amount: 5}"},
+                {"t": 5, "v": "{amount: 2}"},
+                {"t": 13, "v": "{amount: 13}"},
+                {"t": 18, "v": "{amount: 4}"}
+            ]
+        ]
+    }
+]
+....
+
+Events must be loaded into a table before they can be queried.
+You can load events into a table from various data sources, for example files formatted as Parquet or CSV, or by reading events from a stream.
```

Comment:
For streams, I think this works for now, but we may want to change this in the future.  I'm not sure how to word it, but something along the lines of: 

All input data to Kaskada is defined by tables. A table can be backed by files loaded into the system, by a stream source, (or by other external sources like Apache Iceberg).

For a file-backed table, events must be loaded before they can be queried.  For a stream-backed table, Kaskada will automatically include new events in the output calculations as they arrive.

Reply:
I think everything should just be a table.  We will probably have tables that aren't manually loaded (like iceberg backed tables).

Also I could see a future where a table is made up both files and a stream: historical files plus a stream of new data.